nodes/geometry/bmesh_tesselate.py

```diff
@@ -0,0 +1,43 @@
+import bpy
+from bpy.props import *
+from ... events import propertyChanged
+from ... base_types.node import AnimationNode
+
+class BMeshTessellateNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_BMeshTessellateNode"
+    bl_label = "Triangulate BMesh"
+    bl_width_default = 160
+    searchLabels = ["Tesselate BMesh"]
+
+    quad = IntProperty(name = "Quad Method", update = propertyChanged, max = 3, min = 0)
+    ngon = IntProperty(name = "Ngon Method", update = propertyChanged, max = 1, min = 0)
+
+    def create(self):
+        self.newInput("BMesh", "BMesh", "bm")
```

Comment:
![bmesh copy](https://cloud.githubusercontent.com/assets/5881326/15636138/c9837738-25f5-11e6-934e-7163c7497510.PNG)
 dataIsModified has to be set to True.